qubes/tools/qubes_lvm.py

```diff
@@ -0,0 +1,272 @@
+#!/usr/bin/python2
+#
+# The Qubes OS Project, http://www.qubes-os.org
+#
+# Copyright (C) 2016 Bahtiar `kalkin-` Gadimov <bahtiar@gadimov.de>
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+#
+''' Manage pools and volumes managed by the 'lvm_thin' driver. '''
+
+from __future__ import print_function
+
+import datetime
+import logging
+import os
+import subprocess
+import sys
+import time
+import lvm  # pylint: disable=import-error
```

Comment:
I think we don't need it at all. because lvm2 should be installed as dependency by rpm. It was only for my local installation, because my DevVM had no lvm2 installed